.gitlab/tidy-dependabot-pr.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# NOTE: this script is meant to be run on the GitLab CI, it depends on GitLab CI variables
+# Based on https://github.com/signalfx/splunk-otel-java/blob/c9134906c84e9a32a974dec4b380453fe1757410/scripts/propagate-version.sh
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# shellcheck source-path=SCRIPTDIR
```

Comment:
Is this meant to be deleted? Or kept uncommented?

Reply:
Okay, it looks like it's meant to have the `#` at the beginning of the line, it's not just a commented out piece of code. Thanks!